lib/main_common.pm

```diff
@@ -471,6 +475,16 @@ sub load_extra_tests {
     # setup $serialdev permission and so on
     loadtest "console/consoletest_setup";
     loadtest "console/hostname";
+    # post registration testsuites using suseconnect or yast
+    if (have_scc_repos()) {
+       if (get_var("USE_SUSECONNECT")) {
+            loadtest "console/suseconnect_scc";
+       }
+       else {
+           loadtest "console/yast_scc";
+       }
+       return 1;
+    }
```

Comment:
I recommend to not put any scc stuff in lib/main_common.pm which is meant for common code shared by SLE and openSUSE. SCC is clearly (not yet) related to openSUSE. Better make it an explicit test plan in products/sle/main.pm loading just the modules you need. There should be enough helpful examples, e.g. look for "ssh_key_import" in products/sle/main.pm and add an corresponding section.